pkg/new-kernel/patches-5.15.x/0032-rollback-rsb.patch

```diff
@@ -0,0 +1,46 @@
+From 409dfdd8ad3dfc0f69ca1abbfad3d955b00f6a01 Mon Sep 17 00:00:00 2001
+From: Ukhellfire <afaulkner420@gmail.com>
+Date: Fri, 1 Apr 2022 09:44:19 +0100
+Subject: [PATCH] Rollback to r_i2c
+
+---
+ .../dts/allwinner/sun50i-h6-orangepi-3-lts.dts   | 16 +++-------------
+ 1 file changed, 3 insertions(+), 13 deletions(-)
+
+diff --git a/arch/arm64/boot/dts/allwinner/sun50i-h6-orangepi-3-lts.dts b/arch/arm64/boot/dts/allwinner/sun50i-h6-orangepi-3-lts.dts
+index 83fa87039..0b07f8ca2 100644
+--- a/arch/arm64/boot/dts/allwinner/sun50i-h6-orangepi-3-lts.dts
++++ b/arch/arm64/boot/dts/allwinner/sun50i-h6-orangepi-3-lts.dts
+@@ -180,16 +180,6 @@
+ 	bus-width = <4>;
+ 	non-removable;
+ 	status = "okay";
+-
```

Comment:
You are rolling back changes of [0031-add-uwe-bsp-to-orangepi3-lts-dts-file.patch](https://github.com/lf-edge/eve/pull/3088/commits/9f6294e07dbaf8a0c7473840b39b9668a475a6d2#diff-5b0159cfd93488aa31059c5ea79b4ea3e938aa1f618168e0dd4e083763273267) (the previous patch), so there is no reason to keep both patches, just adjust this one and let a note in the commit message...

Reply:
I left all the patches "as is" in armbian, so that it would be more convenient to track the changes later.

Reply:
I see. Another option is to squash some of them and put the commit IDs in the commit message (for tracking).